synapsis/src/main/java/com/soen/synapsis/appuser/AppUserService.java

```diff
@@ -329,6 +334,12 @@ public void markCompanyAsNonVerified(AppUser appUser) {
                 appUserRepository.save(recruiter);
             }
         }
+
+        List<Job> jobsToDelete = jobRepository.findJobsByCompanyEquals(appUser);
+        for (Job jobToDelete : jobsToDelete) {
+            jobRepository.delete(jobToDelete);
+        }
```

Comment:
You can replace the for loop with just `jobRepository.deleteAll(jobsToDelete);`

Reply:
I didn't implement that change, I'm not sure how it's supposed to be.  I'll replace the loop.